test/deparse_tests.c

```diff
@@ -392,6 +392,7 @@ const char* tests[] = {
   "CREATE INDEX \"foo.index\" ON foo USING btree (bar)",
   "CREATE TABLE distributors (did int, name varchar(40), UNIQUE (name) WITH (fillfactor=70)) WITH (fillfactor=70)",
   "SHOW ALL",
+  "CREATE PROCEDURE insert_data(a int, b int) LANGUAGE sql BEGIN ATOMIC INSERT INTO tbl VALUES (a); INSERT INTO tbl VALUES (b); END",
```

Comment:
It might also be good to add these three extra test cases:

```sql
CREATE PROCEDURE do_nothing() LANGUAGE sql BEGIN ATOMIC END
```
```sql
CREATE PROCEDURE returns_one() LANGUAGE sql BEGIN ATOMIC RETURN 1 END
```
```sql
CREATE PROCEDURE updates_and_returns_one() LANGUAGE sql BEGIN ATOMIC UPDATE tbl SET a = 1; RETURN 1 END
```

Reply:
This rules were added.